ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-ref.yaml

```diff
@@ -1,6 +1,7 @@
 ref:
   as: baremetalds-packet-setup
   from: dev-scripts
+  grace_period: 1m0s
```

Comment:
What's this for?

Reply:
I guess if the pod is killed it to ensure the trap doesn't prevent it exiting indefinitely